rdt/transformers/base.py

```diff
@@ -52,6 +164,28 @@ def reverse_transform(self, data):
             data (pandas.Series or numpy.array):
                 Data to transform.
 
+        Returns:
+            pandas.Series:
+                Reverted data.
+        """
+        output_types = self.get_output_types()
+        output_columns = list(output_types.keys())
+        columns_data = data[output_columns]
+        reversed_columns = self._reverse_transform(columns_data)
+        data[self._columns] = reversed_columns
```

Comment:
I'm not sure what our preference stylistically is, but these lines can probably be reduced by saying something like
```
columns_data = data[list(self.get_output_types())]
data[self._columns] = self._reverse_transform(columns_data)
```

Reply:
1) If we go with @csala suggestion to regarding the `get_output_types()`, I think we will always have to append something, yes.

2) Those are the lines @csala wrote in our meeting Friday, which is why I left them like that. I could simplify things a bit though.

Reply:
Good suggestion @amontanez24 Yes, I agree that some lines can be merged here!